### ISSUE TITLE
[Bug]: Cannot find module "esm" required in "consola" package

### ISSUE THREAD

**Issue by le0nik** [nuxt/webpackbar#6]
Error:
```
Error: Cannot find module 'esm'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Volumes/Workspace/bauxy-frontend/node_modules/consola/index.js:5:20)
    at Module._compile (module.js:652:30)
```

Culprit line: https://github.com/nuxt/consola/blob/d4261c246af79deebe494cc2aeecf37909b475ec/index.js#L5

`esm` is listed in "devDependencies" in "consola" package

**Comment by le0nik** [comment]
Hm. Ok then.

**Comment by pi0** [comment]
@le0nik is your problem fixed now?

**Comment by pi0** [comment]
Thank you @le0nik for the report. Actually, it was a packaging mistake. Should be fixed with `webpackbar@2.3.2`
